scripts/build_twilio_library.py

```diff
@@ -30,7 +30,10 @@ def generate(openapi_spec_path: str, output_path: str, language: str, domain: st
 
     to_generate = 'terraform-provider-twilio' if language == 'terraform' else f'twilio-{language}'
     is_domain_irrelevant = language in {'go', 'terraform'} and domain_name == 'preview'
-    sub_dir = subdirectories.get(language, 'rest')
+    if language == 'csharp':
+        sub_dir = subdirectories.get(language, 'Rest')
+    else:
+        sub_dir = subdirectories.get(language, 'rest')
```

Comment:
Ruby and php also has 'Rest', Although those are not yet developed yet.

Reply:
Removed this logic as per first comment